src/stream_base.cc

```diff
@@ -34,6 +35,8 @@ template void StreamBase::AddMethods<StreamWrap>(Environment* env,
                                                  Handle<FunctionTemplate> t);
 template void StreamBase::AddMethods<TLSWrap>(Environment* env,
                                               Handle<FunctionTemplate> t);
+template void StreamBase::AddMethods<JSStream>(Environment* env,
```

Comment:
Btw, this is a good blocker for making this API public. Currently all child classes must be listed here. I guess we can move all these methods to `stream_base.h`, thoughts @bnoordhuis ?